src/lib.rs

```diff
@@ -1,14 +1,14 @@
 // Copyright 2023 the Vello Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-//! Append a [`usvg::Tree`] to a Vello [`Scene`]
+//! Render a [`usvg::Tree`] to a Vello [`Scene`](https://docs.rs/vello/*/vello/struct.Scene.html)
```

Comment:
I would think that this should still use an intra-doc-link?

What was broken about this?

Reply:
Good callout! Fixed.